src/app/fsat/fan-field-data/fan-field-data.component.html

```diff
@@ -83,19 +86,22 @@ <h3>Field Data</h3>
       <div class="input-group">
         <input name="outletPressure" type="number" min="0.1" step="any" class="form-control"
           formControlName="outletPressure" id="{{'outletPressure_'+idString}}" (input)="save()"
-          (focus)="focusField('outletPressure')" onfocus="this.select();" [readonly]="!selected">
+          (focus)="focusField('outletPressure')" onfocus="this.select();" [readonly]="!selected" (input)="compareValues()">
         <span class="input-group-addon units" [innerHTML]="settings.fanPressureMeasurement | settingsLabel"></span>
       </div>
       <span class="alert-danger pull-right small"
         *ngIf="fieldDataForm.controls.outletPressure.invalid && !fieldDataForm.controls.outletPressure.pristine">
         <span *ngIf="fieldDataForm.controls.outletPressure.errors.required">Value Required</span>
-        <span *ngIf="fieldDataForm.controls.outletPressure.errors.min">Value can't be less than
-          {{fieldDataForm.controls.outletPressure.errors.min.min}} <span
-            [innerHTML]="settings.fanPressureMeasurement | settingsLabel"></span></span>
+        <div *ngIf="compareError">
```

Comment:
this should be a part of the form validation. Should be an error on outletPressure